tests/bluetooth/df/common/include/bt_common.h

```diff
@@ -4,4 +4,5 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-void ut_bt_setup(void);
+void *ut_bt_setup(void);
```

Comment:
What is the purpose of modifying the return value if we always return `NULL`?

Reply:
The change is caused by ZTEST framework, signature of a suite and unit tests setup, teardown functions has changed.

Reply:
Roger that :)